<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContrastDialog</class>
 <widget class="QDialog" name="ContrastDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>对比度调节</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	background-color:rgb(65, 65, 65);
}
QDialog QLabel
{
	color:rgb(221, 221, 221);
	qproperty-alignment: AlignCenter;
}
QPushButton
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 6px;
	color:rgb(221, 221, 221);
	padding: 2px 5px;
}
QPushButton:hover
{
	border:2px groove rgb(221, 221, 221);
	color: rgb(83, 83, 83);
	background-color: rgb(221, 221, 221);
}
QPushButton:pressed
{
	border:2px groove rgb(70, 160, 245);
	color: rgb(221, 221, 221);
	background-color: rgb(70, 160, 245);
}
QRadioButton
{
	color:rgb(221, 221, 221);
}
</string>
  </property>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>300</width>
     <height>170</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="linearButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>线性</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="expButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>指数</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="logButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>对数</string>
   </property>
  </widget>
  <widget class="QSlider" name="leftSlopeSlider">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>80</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="leftPointSlider">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>120</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="middleSlopeSlider">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>160</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="rightPointSlider">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>200</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="rightSlopeSlider">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="leftSlopeLabel">
   <property name="geometry">
    <rect>
     <x>388</x>
     <y>65</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>左斜率：</string>
   </property>
  </widget>
  <widget class="QLabel" name="leftPointLabel">
   <property name="geometry">
    <rect>
     <x>388</x>
     <y>105</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>左交点：</string>
   </property>
  </widget>
  <widget class="QLabel" name="middleSlopeLabel">
   <property name="geometry">
    <rect>
     <x>388</x>
     <y>145</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>中斜率：</string>
   </property>
  </widget>
  <widget class="QLabel" name="rightPointLabel">
   <property name="geometry">
    <rect>
     <x>388</x>
     <y>185</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>右交点：</string>
   </property>
  </widget>
  <widget class="QLabel" name="rightSlopeLabel">
   <property name="geometry">
    <rect>
     <x>388</x>
     <y>225</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>右斜率：</string>
   </property>
  </widget>
  <widget class="QSlider" name="expSlider">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="logSlider">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="explogLabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>系数：</string>
   </property>
  </widget>
  <widget class="QLabel" name="leftSlopeValue">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>83</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="leftPointValue">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>123</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="middleSlopeValue">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>163</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="rightPointValue">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>203</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="rightSlopeValue">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>243</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="explogValue">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>93</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="conformButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>310</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="equalizationButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>直 方 图 均 衡 化</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
